benchmark_runner/jupyterlab/templates/analyze_workloads/visualize_workload_operations.py

```diff
@@ -0,0 +1,130 @@
+
+# display df
+from IPython.display import clear_output
+
+# display bokeh
+from bokeh.plotting import figure, show
+from bokeh.io import output_notebook, output_file
+from bokeh.models import HoverTool, Label, ColumnDataSource, LabelSet, Circle, NumeralTickFormatter
+
+
+class VisualizeWorkloadOperations:
+    """
+    This class for visualize workloads operations
+    """
+
+    def __init__(self):
+        self.clear_output()
+        self.width = 1200
```

Comment:
Declare above init method

WIDTH = 1200
HEIGHT = 600

Reply:
Right, fix it